### DISCUSSION TITLE
Configuration on a "per request" basis

### DISCUSSION THREAD

**Discussion by nhh** [directus/directus#16829] [Archive]
I would like to "configure" directus on a per request basis.

For example defining configurations like database url or sqlite databse based on a custom "per request" resolver.

This would be useful to implement multi tenancy with isolation.

Can someone point me to a good starting point? Is this possible to implement atm? I would offer ressources from myself

**Comment by rijkvanzanten** [comment]
There's currently no good way to enable this, as database connections etc are being kept alive for performance reasons. In order to execute this now, you'd almost have to cold start the API for every incoming request, but that comes with a huge performance overhead.
